styleguide/source/_patterns/01-molecules/06-components/22-article-preview/02-article-preview-small/article-preview-small.twig

```diff
@@ -0,0 +1,7 @@
+<div class="topic-articles_article-preview">
```

Comment:
- [ ] create a new base molecule for 'article-preview' that contains all the possible fields. This doesn't actually get included in any patterns, but instead serves as a basis for new variants to be copied from (copy what you want from it into the new twig template). Has the classes already set up for the child elements, which stay the same across variants.
- [ ] style article preview molecules by using the parent class for the pattern (e.g. `.topic_hero` TBD)